addon/lib/metrics.js

```diff
@@ -185,7 +156,7 @@ const Metrics = module.exports = {
     };
 
     TelemetryController.submitExternalPing(
-      pingType, payload,
```

Comment:
This is for the experiment ping, which I didn't really touch in this PR, noted in the pull request description. The only changes here were moving the `pingType` into a constant to match the style of the main add-on ping.